schedule/tests.py

```diff
@@ -61,6 +61,151 @@ def init_course2(self):
             self.course2 = Course.objects.create(**course_attributes)
             alg_course_offering = course_to_course_offering(self.course2)
             add_course_offering_to_schedule(self.course, alg_course_offering)
+    
+    def init_prof(self):
+        #build AppUser instance
+        self.user_attributes = {
+            'username': 'johnd1',
+            'password': 'securepass2',
+            'first_name': 'John',
+            'last_name': 'Doe',
+            'email': 'johnd123@uvic.ca',
+            'is_superuser': False
+        }
+        self.user = User.objects.create_user(**self.user_attributes)
+
+        self.app_user_attributes = {
+            'user': self.user,
+            'prof_type': 'TP',
+            'is_peng': True,
+            'is_form_submitted': True
+        }
+        self.app_user = AppUser.objects.create(**self.app_user_attributes)
+        self.app_user_serializer = AppUserSerializer(instance=self.app_user)
+
+        #create associated Preferences record
+        self.preferences_attributes = {
+            "professor": self.app_user,
+            "is_submitted": True,
+            "taking_sabbatical": False,
+            "sabbatical_length": "NONE",
+            "sabbatical_start_month": 0,
+            "preferred_times": 
+                {"fall": 
+                    {"friday": 
+                        [["10:00", "11:00"],
```

Comment:
sorry this is ugly. I need enough times for algorithms to be able to schedule them and it's what we get from FE.